### ISSUE TITLE
Selection of multi-items

### ISSUE THREAD

**Issue by cathsign** [ludobox/ludobox#78]
We need to precise the trick : press ctrl to select several items in a list

**Comment by MaryPek** [comment]
Yes or is it possible to have some checkbox ?

**Comment by clemsos** [comment]
Yep, so far everything is default HTML5 multiple select. We may want sth more elaborate but that will come with having proper UX design and UI components. 

What I have in mind is using Material UI which have all components already designed and working with React. There is an example here of multiple select  : http://www.material-ui.com/#/components/select-field
